app/canvas/canvas.tsx

```diff
@@ -98,6 +100,8 @@ type DesignModeProps = {
   project: Project;
 };
 
+const dndOptions = { enableMouseEvents: true };
+
```

Comment:
Nitpicking, but why did you decide to extract this? It doesn't seem to be used anywhere else...

Reply:
I see. It makes sense to add a comment about the referential equality then although I’m still not sure it’s the right type of optimization tbh.

Reply:
And no, we shouldn’t use `useMemo`:https://twitter.com/dan_abramov/status/1536518614110679040?s=21&t=ImZzGzodt9XREEPRT30AvA